debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/transforms/MongoDataConverter.java

```diff
@@ -50,10 +50,12 @@ public void convertFieldValue(Entry<String, BsonValue> keyvalueforStruct, Struct
         Object colValue = null;
         String key = keyvalueforStruct.getKey();
         BsonType type = keyvalueforStruct.getValue().getBsonType();
+
+        if (type == BsonType.NULL) {
+            LOG.warn("Field of type NULL is not added to the struct");
```

Comment:
Why is it you don't propagate the null value instead? Is it because we wouldn't know which schema type to use?

Reply:
Exactly. Avro knows `null` type but SchemaBuilder does not.